app/services/cart_service.rb

```diff
@@ -0,0 +1,13 @@
+class Cart_Service
```

Comment:
Os padrão dos nomes das classes, CartService (Camel case), e o objeto por exemplo: cart_service (Snake case)
```suggestion
class CartService
```